var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -540,7 +546,7 @@ def cmake_args(self):
             ):
                 with open(".".join([os.environ["SPACK_CXX"], "cfg"]), "r") as f:
                     options.append("-DCMAKE_CXX_FLAGS={}".format(f.read()))
-            else:
+            elif self.spec.satisfies("^gcc"):
```

Comment:
```suggestion
            elif self.spec.satisfies("%gcc"):
```
I think I got this wrong!

Reply:
No, the relevant case is where we *compiled* with `%oneapi` and compiled a gcc (somehow) to get a satisfactory `gcc-runtime` from it. So it's definitely `+gcc` or `^gcc`

Reply:
I'd like to write the package to say "if you haven't got a satisfactory `gcc-runtime` right now, go compile a `gcc`" but I don't think there is a way to do it.